raphtory/src/algorithms/pathing/dijkstra.rs

```diff
@@ -207,7 +209,13 @@ pub fn dijkstra_single_source_shortest_paths<G: StaticGraphViewOps, T: AsNodeRef
             }
         }
     }
-    Ok(paths)
+    let results_type = std::any::type_name::<(f64, Vec<String>)>();
```

Comment:
I was having a lot of trouble getting it to work as an AlgorithmResult. Also, since the first element of the tuple is just supposed to represent a generic (optionally weighted) distance to the target vertex, it didn't really make sense to tie it to a property type as opposed to a generic numeric representation.